Array/ArrayHomework3.php

```diff
@@ -12,7 +12,6 @@
 for ($i = 0; $i < 3; $i++) {
     for ($j = 0; $j < 5; $j++) {
```

Comment:
There are still number `3` and `5`, you counted elements in your mind :)
Please use `count` or `sizeof`